fastify.js

```diff
@@ -378,6 +383,26 @@ function build (options) {
     return instancePrefix + pluginPrefix
   }
 
+  function cascadeHooksAndMiddlewares (instance) {
+    const parentHooks = instance._hooks
+    const children = instance._children
+
+    for (var i = 0; i < children.length; i++) {
+      const childInstance = children[i]
+      const hooks = childInstance._hooks
+
+      hooks.onRequest.unshift.apply(hooks.onRequest, parentHooks.onRequest)
+      hooks.preHandler.unshift.apply(hooks.preHandler, parentHooks.preHandler)
+      hooks.onSend.unshift.apply(hooks.onSend, parentHooks.onSend)
+      hooks.onResponse.unshift.apply(hooks.onResponse, parentHooks.onResponse)
```

Comment:
can you move this block on a method of Hooks?

Reply:
done 👍